RELEASE_PROCESS.MD

```diff
@@ -0,0 +1,21 @@
+# How We Release Software
+
+## Release on every contribution
+
+Smaller changes incur less risk, and releasing after each contribution ensures we get the changes to users and contributors as quickly as possible. We aim to ship new cookbook releases after any bugfix or feature enhancements are merged. For cookbooks where automated testing has not been implemented there be some delay, as manual testing is necessary.
+
+## Follow Semver
+
+Infrastructure stability for our users is critical so we aim to release predictably versioned cookbooks. Following Semver allows users to pin cookbook requirements in wrapper cookbooks or environment files, and avoid breaking changes. See <http://semver.org/>
+
+## Test Before Releasing
+
+Many cookbooks have integration tests that run using kitchen-docker/kitchen-dokken in Travis CI.  For these cookbooks we're preforming integration testing on every commit. For others integration tests must be run locally and this should be done before any release to ensure high quality releases.
+
+## Always update the changelog
```

Comment:
We want to be clear here that we don't mean for folks contributing to do this. :) we say don't do this, and here we are saying "Always update"